main.tf

```
@@ -55,13 +59,41 @@ resource "aws_vpn_connection" "default" {
   tunnel2_phase1_integrity_algorithms  = var.vpn_connection_tunnel2_phase1_integrity_algorithms
   tunnel2_phase2_integrity_algorithms  = var.vpn_connection_tunnel2_phase2_integrity_algorithms
 
+  tunnel1_log_options {
+    cloudwatch_log_options {
+      log_enabled = var.vpn_tunnel_logging
+      log_group_arn = aws_cloudwatch_log_group.tunnel1[0].arn
+      log_output_format = "json"
+    }
+  }
+
+  tunnel2_log_options {
+    cloudwatch_log_options {
+      log_enabled = var.vpn_tunnel_logging
+      log_group_arn = aws_cloudwatch_log_group.tunnel2[0].arn
+      log_output_format = "json"
+    }
+  }
+
   tags = module.this.tags
 }
 
+resource "aws_cloudwatch_log_group" "tunnel1" {
+  count = local.enabled && var.vpn_tunnel_logging ? 1 : 0
+  name = "/aws/vpn/${terraform.workspace}-${var.name}-tunnel1"
```

Using terraform.workspace here is a no go. Not all are using workspaces, so this will break for folks. Use `module.this.id` here like so: 

```suggestion
  name = "/aws/vpn/${module.this.id}-tunnel1"
```